### PR TITLE
Update Spearhead 1944 (SPE) compatibility, Add SPEX compat

### PR DESCRIPTION
**Pull request by Drofseh**
**When merged this pull request will:**
Improve SPE compat
- Add support for new Mortain terrain
- Add PRC-77 to Milice radio backpack
- Add custom radio configs to new armoured cars, tanks, tank destroyers, etc.
- Fix some small issues with configs and stringtable

Add SPEX compat
- Add support for Utah Beach and Carentan terrains

### CODE REVIEW

File: addons/compat_spe/SPE_Tanks.hpp [hunk 1]
```diff
@@ -354,9 +390,153 @@ class SPE_FR_M4A0_75_mid_Command: SPE_FR_M4A0_75_mid {
     };
 };
 class SPE_FR_M4A0_75_mid_Command_DVL: SPE_FR_M4A0_75_mid {
+    editorSubCategory = "SPE_EdSubcat_Tanks_DLV";
+    hasDriver = -1;
     vehicleClass = "Armored_DLV";
+};
+
+class SPE_M4A0_105: SPE_M4A0_75 {
```
Comment (PabstMirror) [inline addons/compat_spe/SPE_Tanks.hpp]:
```suggestion
class SPE_M4A0_105: SPE_M4A0_75 {};
```

Comment (Drofseh) [reply addons/compat_spe/SPE_Tanks.hpp]: Good spot, but we actually don't need to the inheritence at all here!
I'll update it to be `class SPE_M4A0_105;`

**Review by PabstMirror — APPROVED** [review]
configs look fine

I'm not thrilled about adding new vehicles in a compat,
but I think I understand why it's being done

**Comment by Drofseh** [comment]
> configs look fine
> 
> I'm not thrilled about adding new vehicles in a compat, but I think I understand why it's being done

The new vehicles should be in their 1.1.1 update, they didn't have time for them in 1.1

**Review by jonpas — APPROVED** [review]

**Comment by jonpas** [comment]
@Drofseh Needs conflict resolution, then good to merge.

**Comment by Drofseh** [comment]
@jonpas deconflicted

**Comment by Drofseh** [comment]
@jonpas or @PabstMirror can this be merged?
It's been a few weeks now since SPE updated and the new terrain isn't supported until this PR is released.

**Review by PabstMirror — APPROVED** [review]

**Review by jonpas — APPROVED** [review]

**Pull request merged**